pom.xml

```diff
@@ -347,7 +347,7 @@
             <configuration>
               <classifier>${javacpp.platform}${javacpp.platform.extension}</classifier>
               <skipIfEmpty>true</skipIfEmpty>
-              <includes> 
```

Comment:
Let's also revert this, even if it shouldn't be there, to help with any merge conflicts later...